test/freenet/support/MultiValueTableTest.java

```diff
@@ -218,8 +217,8 @@ public void testCountAll() {
 		Object[][] sampleObjects = 
 			createSampleKeyMultiVal(sampleKeyNumber,sampleMaxValueNumber,sampleIsRandom);
 		methodMVTable = fillMultiValueTable(sampleObjects);
-		for(int i=0;i<sampleObjects.length;i++)
-			assertEquals(((List<Object>)(sampleObjects[i][1])).size(),methodMVTable.countAll(sampleObjects[i][0]));
+		for (Object[] sampleObject : sampleObjects)
```

Comment:
Cast `sampleObjects`, change the type of `sampleObject`, remove cast in line below. And braces, please.